pyarc2stac/utils.py

```diff
@@ -22,6 +22,8 @@ def convert_to_datetime(times_extent):
        datetime.fromtimestamp((time_extent/1000.0), timezone.utc) for time_extent in times_extent
     ]
 
+def rfc3339_format(self):
```

Comment:
I think this project also uses the pystac package which has a rfc3339 method which might be handy. The string formatting in this PR looks right, too, though. 

https://pystac.readthedocs.io/en/latest/api/utils.html#pystac.utils.datetime_to_str

Reply:
yes, let's use what's already available!